core/src/main/java/io/ultrabrew/metrics/data/BasicHistogramAggregator.java

```diff
@@ -82,8 +81,7 @@ public BasicHistogramAggregator(final String metricId, final DistributionBucket
    */
   public BasicHistogramAggregator(final String metricId, final DistributionBucket buckets,
```

Comment:
`metricId` should probably be renamed `metricName` since it's not an internal id but the actual name from the user.

Reply:
Do not change the public interface with backwards incompatible changes. Also identifier was chosen instead of name for a reason as it is an identifier, not just a label